### PR TITLE
7.0.x Change converter to suit ballerina 0.89 version

### PR DESCRIPTION
**Pull request by arukshani**
1)logger changes
2)include protocol name for service
3) change service annotations
4) Initial database commit

### CODE REVIEW

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java [hunk 1]
```diff
@@ -192,19 +194,30 @@ private String getElementOrAttributeName(QName qName) {
...
      *
      * @param attributes List of attributes associate with a mule element
-     * @param mClass     Intermediate class that is mapped to the mule element
+     * @param mClassName Intermediate class that is mapped to the mule element
      */
-    private void populateIntermediateObject(Iterator<Attribute> attributes, Class<?> mClass) {
+    private void populateIntermediateObject(Iterator<Attribute> attributes, String mClassName, String elementName) {
         try {
-            java.lang.Object object = mClass.newInstance();
+            java.lang.Object object = null;
```
Comment (pamod) [inline tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java]: Any reason for using fully qualified classes here ?

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java [hunk 2]
```diff
@@ -214,14 +227,37 @@ private void populateIntermediateObject(Iterator<Attribute> attributes, Class<?>
...
+            if (!(baseObj instanceof Database)) {
+                baseObj.buildTree(dataCarrierDTO);
+            }
 
         } catch (IllegalAccessException e) {
             logger.error(e.getMessage(), e);
         } catch (InstantiationException e) {
             logger.error(e.getMessage(), e);
+        } catch (ClassNotFoundException e) {
+            logger.error(e.getMessage(), e);
```
Comment (pamod) [inline tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java]: Any reason for not throwing the exception further ?

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java (hunk 1 above)
Comment (arukshani) [reply tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java]: Replaced with import. https://github.com/wso2/product-ei/pull/1216/commits/5bc332461335b908d9a95df22ffdb43bd8ebe7c0

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java (hunk 2 above)
Comment (arukshani) [reply tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java]: At this point even if this type of an error occurred conversion process should not halt. and proper messages were added in here >> https://github.com/arukshani/product-ei/commit/5bc332461335b908d9a95df22ffdb43bd8ebe7c0#diff-ea4cf52b0ef514be466d6bf957833ba3L226

**Pull request merged**